### DISCUSSION TITLE
Any plan for supporting one command line install tailwindcss and React?

### DISCUSSION THREAD

**Discussion by zning** [tailwindlabs/tailwindcss#4244] [Ideas]
I'd like to use Tailwindcss with React but didn't like all the configurations. Do you guys have any plan for one script npm/yarn installation for easy setup? Possibly something like "npx create-tailwindcss-react-app" ?? Or even with storybook. 
I guess since many people using just a few combinations, why not just rebuild them for easy implementations

**Comment by nonissue** [comment]
I mean, these already exist. I did a quick google and found several, eg:

https://github.com/GeoffSelby/cra-template-tailwindcss

That said, it's not really the best way to go IMO. The tailwind docs have instructions on how to setup create react app and tailwind together, and I think it's best to follow the guide so you have some idea of how tailwind works.

**Comment by zning** [comment]
I found this as well but reluctant to use it for many reasons. I personally think official scripts is more secure and trustworthy. Can I follow the instructions to install that manually, yes. But do I want to do that every time and make sure everything is not broken, probably not. Still remember back these old days when everything need to be installed and setup manually even for react. With the create-react-app came up, I never want to go back to setup myself. The Manuel way is still good if you have an existing project. But if you do lots of testing and setups, nothing beat a simple one line commands. Besides, it attracts more people adopted this great library and want to try it out.
